plugins/cpp_metrics/model/include/model/cppdirectorymetrics.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CC_MODEL_CPPDIRECTORYMETRICS_H
+#define CC_MODEL_CPPDIRECTORYMETRICS_H
+
+#include <model/file.h>
+
+namespace cc
+{
+namespace model
+{
+
+#pragma db object
+struct CppDirectoryMetrics
```

Comment:
Could be `CppFileMetrics` and we ca handle files and directories in a uniform way like the classic *metrics* plugin.

Reply:
Fixed in a904b68c9f6b79f09bb04a1a1461cec0d803af9e.

Reply:
It was not fixed, it should still be renamed to `CppFileMetrics`.